pkg/cli/image_cmd_test.go

```diff
@@ -376,7 +376,20 @@ func TestSignature(t *testing.T) {
 		str := space.ReplaceAllString(buff.String(), " ")
 		actual := strings.TrimSpace(str)
 		So(actual, ShouldContainSubstring, "IMAGE NAME TAG DIGEST SIGNED SIZE")
-		So(actual, ShouldContainSubstring, "repo7 test:1.0 883fc0c5 true")
+		So(actual, ShouldContainSubstring, "repo7 test:1.0 883fc0c5 true 15B")
```

Comment:
why the size was not shown before?

you didn't modify anything on this part

Reply:
It was shown, but not verified.